Makefile

```diff
@@ -30,13 +30,13 @@ install:
 # local venv.
 build-travis-narrative:
 	bash $(INSTALLER) && \
-	grunt minify && \
+	npx grunt minify && \
 	sed <src/config.json.templ >src/config.json "s/{{ .Env.CONFIG_ENV }}/dev/" && \
 	sed -i 's/{{ if ne .Env.CONFIG_ENV "prod" }} true {{- else }} false {{- end }}/true/' src/config.json && \
 	jupyter notebook --version
 
 test: test-backend test-frontend
-	@echo "done running backend and frontend test scripts"
+	@echo "done running backend and frontend test scripts"n
```

Comment:
Typo?

Reply:
Hmm, finger must have just slipped. That sometimes happens when doing `[Command]s` and the `[Command]` key doesn't take, but "n" I think my finger must have just twitched!